Automattic/jetpack
# Enable restapi tests in multisite configuration

The rest-api tests had an incorrect path and weren't executing in multisite tests. This patch re-enables the tests.

#### Changes proposed in this Pull Request:
* re-enable multisite testing to include rest-api

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
N/A

#### Testing instructions:
Nothing really to check here except to confirm tests run successfully for multisite.

#### Proposed changelog entry for your changes:
N/A
